### PR TITLE
Code review changes from #995

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: build-and-test
+name: test
```
Comment (crobert-1) [inline .github/workflows/test.yml:1]: My apologies, I think build-and-test would be better than test for both. It just seems closer to what's actually happening.

Comment (atoulme) [reply .github/workflows/test.yml:1]:
```suggestion
name: build-and-test
```

Comment (atoulme) [reply .github/workflows/test.yml]: I'll rename the file.

**Review by crobert-1 — APPROVED** [review]
Thanks!

**Pull request merged**